.travis.yml

```diff
@@ -1,14 +1,9 @@
 language: objective-c
-matrix:
-  fast_finish: true
-env:
-  - FORMULA=zeroc-ice35 BREW_OPTS=--with-python
-  - FORMULA=omero BREW_OPTS=--only-dependencies
+env: VERBOSE=1
```

Comment:
Does verbose need adding here, since it's also set in the `scripts` section, and it's likely not useful as a matrix parameter?